valhalla/jawn/src/lib/handlers/LoggingHandler.ts

```diff
@@ -418,4 +458,63 @@ export class LoggingHandler extends AbstractLogHandler {
 
     return requestInsert;
   }
+
+  private pullRequestBodyMessage(requestBody: any): string {
```

Comment:
Nit: This name is a bit confusing maybe `extract` instead of `pull`

```suggestion
  private pullRequestBodyMessage(requestBody: any): string {
```